composer.json

```diff
@@ -75,7 +75,7 @@
       "dealerdirect/phpcodesniffer-composer-installer": true
     },
     "platform": {
-      "php": "5.6.20"
+      "php": "7.0.8"
```

Comment:
Curious why `7.0.8` and not just `7.0.0`?

Reply:
I have used `7.0.8` because one of the `symfony/finder` package requires a minimum PHP version of `7.0.8`

The error with `7.0.0`
```
  Problem 1
    - symfony/finder v3.4.47 requires php ^5.5.9|>=7.0.8 -> your php version (7.0.0; overridden via config.platform, actual: 7.4.29) does not satisfy that requirement.
    - wp-cli/wp-cli v2.6.0 requires symfony/finder >2.7 -> satisfiable by symfony/finder[v3.4.47].
    - wp-cli/wp-cli is locked to version v2.6.0 and an update of this package was not requested.
```